docs/migration/cosmoshub-4-delta-upgrade.md

```diff
@@ -0,0 +1,124 @@
+
+### Current runtime, cosmoshub-4 (pre-Delta upgrade) is running Gaia v4.2.1
+
+The Cosmos Hub mainnet network, `cosmoshub-4`, is currently running [Gaia v4.2.1](https://github.com/cosmos/gaia/releases/tag/v4.2.1). We anticipate that operators who are running earlier versions of Gaia, e.g., v4.2.x, will be able to upgrade successfully; however, this is untested and it is up to operators to ensure that their systems are capable of performing the upgrade. 
+
+### Target runtime, cosmoshub-4 (post-Delta upgrade) will run Gaia v5.0.0
+
+The Comsos Hub mainnet network, `cosmoshub-4`, will run [Gaia v5.0.0](https://github.com/cosmos/gaia/releases/tag/v5.0.0). Operators _MUST_ use this version post-upgrade to remain connected to the network. 
+
+## Delta upgrade steps
+
+The following steps assume that an operator is running v4.2.1 (running an earlier version is untested). The upgrade has only been tested with v4.2.1 and these instructions follow this prerequisite.
+
+1. Prior to the upgrade, operators _MUST_ be running Gaia v4.2.1.
+2. At the upgrade block height of [6910000](#Upgrade-will-take-place-July-12,-2021), the Gaia software will panic with a message similar to the below:
+
+> ERR UPGRADE "Gravity-DEX" NEEDED at height: 6910000: v4.2.1
+> 
+> ERR CONSENSUS FAILURE!!! err="UPGRADE \"Gravity-DEX\" NEEDED at height: 6910000: v4.2.1" module=consensus stack="goroutine ...
+
+3. Replace the Gaia v4.2.1 binary with the Gaia v5.0.0 binary
+4. Start the Gaia v5.0.0 binary using the following command (also applying any additional flags and parameters to the binary needed by the operator, e.g., `--home $HOME`):
+
+> gaiad start --x-crisis-skip-assert-invariants
+
+IMPORTANT: The flag `--x-crisis-skip-assert-invariants` is optional and can be used to reduce memory and processing requirements while the in-place ugprade takes place before resuming connecting to the network.
+
+5. Wait until 2/3+ of voting power has upgraded for the network to start producing blocks
+
+## Upgrade duration
+
+The upgrade may take several hours to complete because cosmoshub-4 participants operate globally with differing operating hours and it may take some time for operators to upgrade their binaries and connect to the network.
+
+## Rollback plan
+
+During the network upgrade, core Cosmos teams will be keeping an ever vigilant eye and communicating with operators on the status of their upgrades. During this time, the core teams will listen to operator needs to determine if the upgrade is experiencing unintended challenges. In the event of unexpected challenges, the core teams, after conferring with operators and attaining social consensus, may choose to declare that the upgrade will be skipped. 
+
+Steps to skip this upgrade proposal are simply to resume the cosmoshub-4 network with the (downgraded) v4.2.1 binary using the following command:
+
+> gaiad start --unsafe-skip-upgrade
```

Comment:
> gaiad start --unsafe-skip-upgrade 6910000

For skip [Proposal #51](https://www.mintscan.io/cosmos/proposals/51) upgrade, I think it would be accurate to specify the height.

Reply:
Thanks! Docs weren't clear that this parameter was required.